gc-worker/cmd/main.go

```diff
@@ -0,0 +1,99 @@
+// Copyright 2022 TiKV Project Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package main
+
+import (
+	"context"
+	"fmt"
+	"gc-worker/server"
+	"gc-worker/server/config"
+	"os"
+	"os/signal"
+	"syscall"
+
+	"github.com/pingcap/log"
+	"go.uber.org/zap"
+)
+
+func main() {
+	cfg := config.NewConfig()
+	err := cfg.Parse(os.Args[1:])
+	if err != nil {
+		log.Fatal("parse cmd flags error", zap.Error(err))
+		fmt.Printf("parse cmd flags error, %v", err)
```

Comment:
should it be removed?

Reply:
i think we can keep it. If input arguments are invalid, we need print the error message in screen and log file both. User may be confused is no message ouput but process exit.

Reply:
ok, should we print the error to stdout first?

Reply:
ok